geode-redis/README.md

```diff
@@ -183,30 +183,30 @@ start server \
+| PERSIST               | SSCAN                                                             | CLIENT TRACKING               	|
+| PEXPIRE               | SUNION                                                            | CLIENT UNBLOCK                	|
+| PEXPIREAT             | SUNIONSTORE                                                       | CLUSTER ADDSLOTS              	|    
+| PING                  | TIME                                                              | CLUSTER BUMPEPOCH                 |             
+| PSUBSCRIBE            | UNLINK [1]                                                        | CLUSTER COUNT-FAILURE-REPORTS 	|                
+| PTTL                  |                                                                   | CLUSTER COUNTKEYSINSLOT       	|                
 | PUBLISH               |                                                                   | CLUSTER DELSLOTS     	            |            
 | PUNSUBSCRIBE          |                                                                   | CLUSTER FAILOVER              	|                
 | QUIT                  |                                                                   | CLUSTER FLUSHSLOTS                |             
 | RENAME                |                                                                  	| CLUSTER FORGET                    | 
 | SADD                  |                                                                  	| CLUSTER GETKEYSINSLOT             | 
 | SET                   |                                                                   | CLUSTER INFO                  	|
 | SETNX              	|                                                                   | CLUSTER KEYSLOT                   |
-| SMEMBERS              |                                                                   | CLUSTER MEET                      | 
-| SREM                  |                                       	                        | CLUSTER MYID                   	|    
-| STRLEN                |                                       	                        | CLUSTER NODES                 	|
-| SUBSCRIBE          	|                                                                   | CLUSTER REPLICAS                  | 
-| TTL                   |                                      	                            | CLUSTER REPLICATE              	|
-| TYPE                  |                                        	                        | CLUSTER RESET                  	|
-| UNSUBSCRIBE           |                                                                   | CLUSTER SAVECONFIG                |
-|                       |             	                                                    | CLUSTER SET-CONFIG-EPOCH          |
+| SLOWLOG               |                                                                   | CLUSTER MEET                      | 
```

Comment:
Maybe we could put a number next to this one like we did for `UNLINK` and put a footer note at the bottom of the file that it's a NoOp.